users/tests/test_api_authentication.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse
+from rest_framework.exceptions import ErrorDetail
+from rest_framework_simplejwt.token_blacklist.models import BlacklistedToken, OutstandingToken
+
+User = get_user_model()
+
+
+@pytest.fixture()
+def login_url() -> str:
```

Comment:
Please change names to contain api in the name, like "api_login_url". It will make it more readable.